blog/test/controllers/articles_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
you're not using these test files so I'd say get rid of them but not yet.  testing will come up